hr/api/models.py

```diff
@@ -0,0 +1,6 @@
+from django.db import models
+
+class EventLog(models.Model):
+    event_data = models.TextField(blank = False)
+    sequence = models.IntegerField(blank = False)
+    guid = models.AutoField(primary_key = True)
```

Comment:
we can use the default pk for this table as its primary key (unique for each event log entry) and create another field to identify which expense it belongs to (which is not unique in this table).